pkgs/top-level/python-packages.nix

```diff
@@ -21199,6 +21199,8 @@ EOF
     };
   };
 
+  eduvpn-client = callPackage ../applications/networking/python-eduvpn-client { };
```

Comment:
If we want this to live inside `pythonPackages`, then `default.nix` would need to be moved to `developments/python-modules/eduvpn-client` and the python dependencies would have to be specified as arguments.
However, I think if the API is not used we shouldn't have it inside `pythonPackages`. The package supporting multiple python versions isn't an arguments because we can simply override `pythonPackages = python3Packages`.

Reply:
@Mic92 What's your opinion?

Reply:
@leenaars You would need to address this before the PR can be merged. Everything that belongs to `python-packages.nix` resides in `development/python-modules`, so you would need to move the expression to `developments/python-modules/eduvpn-client` as suggested above. Other option is to move eduvpn-client back to `all-packages.nix` and keep it at `tools/networking/`, that should be ok only if we assume that nobody is going to use the API from this package.